packages/js/product-editor/src/components/modal-editor/style.scss

```diff
@@ -0,0 +1 @@
+@import './modal-editor.scss';
```

Comment:
I'm curious why we have multiple levels of scss imports here? Do we need both this file and `modal-editor-scss` files to exist?

Reply:
I was following our pattern of having an `index.ts` file with a separate `component-name.tsx` file that is imported there.  It's easier to find and identify stylesheets this way that are named `component-name.scss`.  It also creates an entry file when multiple imports do exist.

That said, I'm not overly opinionated on this and have removed it for the time being.